### ISSUE TITLE
RFC: Make this useful™

### ISSUE THREAD

**Issue by swantzter** [solanum-ircd/solanum.chat#2]
So right now we have a hard coded landing page, that won't do.

There seems to be discussion on making the website home for documentation and such and have links like https://solanum.chat/identify-msg for capabilities work

The easiest way to build something that's also easy to maintain for this would naturally be a static site, be that github pages' default jekyll or other SSG tool like Hugo, 11ty that runs in a github action and pushes to gh-pages...

This issue is in itself a huge one and might need to be broken up into multiple smaller ones but it's a start. I'm opening this up as an RFC with the following questions:

- What should the site do? What should it contain? 
    - docs for capabilities, what should be on them? Name, explanation of what they do, link to spec or spec, link to code?
    - more general docs? Building, contributing?
    - updates/changelog?
    - none of the above, other things?
- how to discover content?
    - basically what is the navigation, is there a page listing all capabilities with a quick summary and a click to show more?
- How should it look?
    - logo, colour theme, fonts etc.

**Comment by edk0** [comment]
my opinions:

* docs for solanum vendor extensions are a must—every name in any `solanum.chat/` ircv3 namespace should resolve to or redirect to a spec for that extension
* other documentation would be nice, but I don't have an opinion about whether it should be on the website
* a tree-like sitemap is probably sufficient to be getting on with
* I want the logo to be a nightshade flower. I know a couple of people have made a start drawing some. If we need a highlight colour some sort of nightshadey purple might be a good bet, but we should keep it simple
